pkgs/applications/virtualization/cri-o/wrapper.nix

```diff
@@ -0,0 +1,51 @@
+{ cri-o-unwrapped
+, runCommand
+, makeWrapper
+, lib
+, extraPackages ? []
+, cri-o
+, runc # Default container runtime
+, crun # Container runtime (default with cgroups v2 for podman/buildah)
+, conmon # Container runtime monitor
+, utillinux # nsenter
+, cni-plugins # not added to path
+, iptables
+}:
```

Comment:
We still need `socat`for port forwarding purposes.

Reply:
Fixed.